docs/common/contact-registration.md

```diff
@@ -0,0 +1,108 @@
+DateReviewed: 2022-12-16
+
+Registering Contact Information
+===============================
+
+OSG staff keep track of contact information for OSG Consortium participants to provide access to OSG services,
+notify administrators and security contacts of software and security updates,
+and coordinating in case of security incidents or troubleshooting services.
+
+The OSG contact management service is backed by [InCommon federation](https://www.incommon.org/federation/),
+meaning that contacts may register with the OSG using their institutional identities with familiar Single Sign-On forms.
+
+!!! info "Privacy Notice"
+    The OSG treats any email addresses and phone numbers as confidential data but does not make any guarantees of
+    privacy.
+    All other data is public (such as name, GitHub username, and any association with particular services or
+    collaborations).
+
+Submitting an Application
+-------------------------
+
+To register with the OSG, submit an application using the self-signup process:
+
+1.  Visit <https://osg-htc.org/register>
+
+1.  You will be presented with a Single-Sign On page.
+    Select your insitution and sign in with your insitutional credentials:
+
+    !!! question "Help, my institution does not show up in the drop-down!"
+        If your institution does not show up in the drop-down menu, then your institution is not part of the
+        [InCommon federation](https://www.incommon.org/federation/).
+        Instead, we recommend using an [ORCID account](https://orcid.org/), registering a new one if necessary.
```

Comment:
To my eye, the leading "Instead" makes it sound unclear whether we _always_ recommend using an OCRID account instead, or just in the case that your institution does not show up.

Does this read any better for you?

```suggestion
        In this case, we recommend using an [ORCID account](https://orcid.org/) instead, registering a new one if necessary.
```